src/pptrees/adder_tree.py

```diff
@@ -102,6 +107,106 @@ def _hdl_preamble(self):
 
         return (preamble,used_modules)
 
+    def _vhdl_preamble(self):
+        """VHDL preamble for the adder's HDL"""
+
+        preamble = ""
```

Comment:
You should probably use a list here and then a 
```python
preamble = []
preamble.append('a')
preamble.append('b')
"\n".join(preamble)
```

Reply:
Not opposed to this at all. Both ways are equal in my eyes.

Which way do you consider more Pythonic?

Reply:
Using `a += "xyz"` is n^2 as it requires copying the whole string each time. See https://stackoverflow.com/questions/34008010/is-the-time-complexity-of-iterative-string-append-actually-on2-or-on

Reply:
Compelling argument!